apps/search/index.js

```diff
@@ -0,0 +1,78 @@
+import express from 'express';
+import SearchBlocks from '../../collections/search_blocks';
+import { data as sd } from 'sharify';
+import { extend } from 'underscore';
+import apolloMiddleware from 'react/apollo/middleware';
+import setModeMiddleware from './middleware/setMode';
+import setSearchMiddleware from './middleware/setSearch';
+import setHeaderMiddleware from './middleware/setHeader';
+
+const app = express();
+
+app.set('views', `${__dirname}/templates`);
+app.set('view engine', 'jade');
+
+const renderSearch = (req, res, next) => {
+  const blocks = new SearchBlocks();
+  const search = res.locals.sd.SEARCH;
+
+  if (req.params.block_id) {
```

Comment:
FYI, we don't support these urls anymore so this is fine to remove.

Reply:
👍